File: docs/source/quickstart.R
# 1-library
library(opendp)
enable_features("contrib")
# 2-use
space <- c(atom_domain(.T = "f64"), absolute_distance(.T = "f64"))
laplace_mechanism <- space |> then_laplace(scale = 1.)
dp_agg <- laplace_mechanism(arg = 23.4)

File: docs/source/quickstart.py
'''
# init
>>> from opendp.mod import enable_features
>>> enable_features('contrib')

# /init

# dp-demo
>>> import opendp.prelude as dp
>>> base_laplace = dp.space_of(float) >> dp.m.then_base_laplace(scale=1.)
>>> dp_agg = base_laplace(23.4)

# /dp-demo
'''
